resource access update (#127)
DavidXanatos/sandboxie-docs: Content/SBIE2111.md

# SBIE2111

**Message:** SBIE2111 Process is not accessible: _program_, call _call_

**Logged To:** [Popup Message Log](PopupMessageLog.md).

**Explanation:**

This is an informational message.

> Before v1.0.16 / 5.55.16, Sandboxie allowed sandboxed programs to read the memory of any unsandboxed program belonging to the current user, this is obviously a bad idea if your goals is not only infection prevention but also data protection. Hence with v1.0.16 / 5.55.16 onwards Sandboxie will not allow for _PROCESS_VM_READ_ on unsandboxed processes or processes belonging to other boxes.

Reviewer: ```suggestion
Before v1.0.16 / 5.55.16, Sandboxie allowed sandboxed programs to read the memory of any unsandboxed program belonging to the current user, this is obviously a bad idea if your goals is not only infection prevention but also data protection. Hence, from v1.0.16 / 5.55.16 onwards Sandboxie will not allow for _PROCESS_VM_READ_ on unsandboxed processes or processes belonging to other boxes.
```

> To facilitate compatibility this build introduces a IPC options, with ```ReadIpcPath=$:program.exe``` any unboxed process can be configured to allow for _PROCESS_VM_READ_, it is also possible to restore the old behavior entirely by specifying ```ReadIpcPath=$:*```.

Reviewer: ```suggestion
To facilitate compatibility, this build introduces a IPC option. With `ReadIpcPath=$:program.exe`, any unboxed process can be configured to allow for _PROCESS_VM_READ_ and it is also possible to restore the old behavior entirely by specifying `ReadIpcPath=$:*`.
```

> By default the only process whos memory can be read is _explorer.exe_ many processes want that and explorer should not keep any secrets normally anyways. To block this you can use ```ClosedIpcPath=$:explorer.exe```.

Reviewer: ```suggestion
By default, the only process whose memory can be read is _explorer.exe_. Many processes require it and Windows File Explorer should not keep any secrets anyway. To block this, you can use `ClosedIpcPath=$:explorer.exe`.
```

> By default, the only process whose memory can be read is _explorer.exe_. Many processes require it and Windows File Explorer should not keep any secrets anyway. To block this, you can use `ClosedIpcPath=$:explorer.exe`.

Reviewer: ```suggestion
To facilitate compatibility, this build introduces a [ReadIpcPath](ReadIpcPath.md) option.
```

This should avoid the repetition of information already provided in the ReadIpcPath file (if you want, you can rewrite it in another way).
